tools/tools.mk

```diff
@@ -16,13 +16,18 @@ PROTOC_GEN_GO_GRPC := $(TOOLS_BIN_DIR)/protoc-gen-go-grpc
 PROTOC_GEN_GO_HASHPB := $(TOOLS_BIN_DIR)/protoc-gen-go-hashpb
 PROTOC_GEN_GO_VTPROTO := $(TOOLS_BIN_DIR)/protoc-gen-go-vtproto
 PROTOC_GEN_GRPC_GATEWAY := $(TOOLS_BIN_DIR)/protoc-gen-grpc-gateway
+PROTOC_GEN_JSONSCHEMA := $(TOOLS_BIN_DIR)/protoc-gen-jsonschema
 PROTOC_GEN_OPENAPIV2 := $(TOOLS_BIN_DIR)/protoc-gen-openapiv2
 PROTOC_GEN_VALIDATE := $(TOOLS_BIN_DIR)/protoc-gen-validate
 
 GEN_DIR := api/genpb
+JSONSCHEMA_DIR := schema/jsonschema
 OPENAPI_DIR := schema/openapiv2
 MOCK_DIR := internal/test/mocks
 
+PROTOC_GEN_JSONSCHEMA_SRC_DIR := hack/tools/protoc-gen-jsonschema
+PROTOC_GEN_JSONSCHEMA_SRC_FILES := go.mod go.sum $(shell find $(PROTOC_GEN_JSONSCHEMA_SRC_DIR) -type f -name '*.go')
```

Comment:
Let's create a separate, nested go module for `hack/tools/protoc-gen-jsonschema` so that it's self-contained. Then you won't need to worry about adding build tags to the files either.

Reply:
Good idea!